### PR TITLE
"login button created"

### PR DESCRIPTION
**Pull request by richardcolachagua**

### CODE REVIEW

File: src/pages/HomePage/Header.jsx [hunk 1]
```diff
@@ -27,13 +27,15 @@ const Header = ({ pricingRef, contactRef, featureRef, onButtonClick }) => {
...
+            <div style={{ display: "flex", alignItems: "center" }}>
               <Tabs textColor="inherit">
                 <Tab label="Features" onClick={featureRef} />
                 <Tab label="Pricing" onClick={pricingRef} />
                 <Tab label="Contact" onClick={contactRef} />
               </Tabs>
-
+              <StyledButton style={{}} onClick={onButtonClick}>
+                Login
+              </StyledButton>
```
Comment (AlanisYates) [inline src/pages/HomePage/Header.jsx:38]: Great! This mirrors the functionality of the signup button. In HomePage.jsx, let's create a new function that redirects the user to the signup page when they click the signup button. Then, we can prop-drill this function to the header component, similar to how onButtonClick is currently handled.

File: src/pages/HomePage/Header.jsx [hunk 2]
```diff
@@ -27,13 +27,15 @@ const Header = ({ pricingRef, contactRef, featureRef, onButtonClick }) => {
...
           <Hidden mdDown>
-            <div style={{ display: 'flex', alignItems: 'center' }}>
+            <div style={{ display: "flex", alignItems: "center" }}>
               <Tabs textColor="inherit">
                 <Tab label="Features" onClick={featureRef} />
                 <Tab label="Pricing" onClick={pricingRef} />
                 <Tab label="Contact" onClick={contactRef} />
               </Tabs>
-
+              <StyledButton style={{}} onClick={onButtonClick}>
```
Comment (AlanisYates) [inline src/pages/HomePage/Header.jsx:36]: Why do you have a style here?

**Review by AlanisYates — CHANGES_REQUESTED** [review]

File: src/pages/HomePage/Header.jsx (hunk 2 above)
Comment (richardcolachagua) [reply src/pages/HomePage/Header.jsx]: removed

File: src/pages/HomePage/Header.jsx (hunk 1 above)
Comment (richardcolachagua) [reply src/pages/HomePage/Header.jsx]: function created

**Comment by AlanisYates** [comment]
This looks good! - resolve the merge conflicts and you can Merge

**Pull request merged**